packages/core-modules/contracts/BaseProxy.sol

```diff
@@ -2,18 +2,9 @@
 pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/proxy/ForwardingProxy.sol";
-import "./storage/ProxyStorage.sol";
 
-contract BaseProxy is ForwardingProxy, ProxyStorage {
+contract BaseProxy is ForwardingProxy {
     constructor(address firstImplementation) {
```

Comment:
BaseProxy is the only proxy that is not going to be behind another... here the constructor works